pkg/bot/utils.go

```diff
@@ -27,3 +27,29 @@ import (
 func formatCodeBlock(msg string) string {
 	return fmt.Sprintf("```\n%s\n```", strings.TrimSpace(msg))
 }
+
+//accessAndTypeCastToMap access and typecast of map object
+func accessAndTypeCastToMap(key string, m map[string]interface{}) (map[string]interface{}, error) {
+	value, ok := m[key]
+	if !ok {
+		return nil, fmt.Errorf("Lark: %s not found", key)
```

Comment:
I believe this is meant to be a generic function. Let's remove `Lark` keyword from the log
```suggestion
		return nil, fmt.Errorf("%s not found in the map", key)
```